src/shared/components/SelectEducationLevelModal/SelectEducationLevelModal.tsx

```diff
@@ -0,0 +1,98 @@
+import './SelectEducationLevelModal.scss';
+
+import {
+	Button,
+	ButtonToolbar,
+	Modal,
+	ModalBody,
+	type ModalProps,
+	Select,
+	Toolbar,
+	ToolbarItem,
+	ToolbarRight,
+} from '@viaa/avo2-components';
+import { type LomFieldSchema } from '@viaa/avo2-types/types/lom';
+import React, { type FunctionComponent, useCallback, useMemo, useState } from 'react';
+
+import { EducationLevelId } from '../../helpers/lom';
+import { tHtml, tText } from '../../helpers/translate';
+import withUser, { type UserProps } from '../../hocs/withUser';
+import { useLomEducationLevels } from '../../hooks/useLomEducationLevels';
+
+type SelectEducationLevelModalProps = Omit<ModalProps, 'children'> &
+	Partial<UserProps> & {
+		onConfirm?: (lom: LomFieldSchema) => void;
+	};
+
+// Component
+
+const SelectEducationLevelModal: FunctionComponent<SelectEducationLevelModalProps> = (props) => {
+	const { user, commonUser, onConfirm, ...modal } = props;
+
+	const [educationLevels] = useLomEducationLevels();
+	const [selected, setSelected] = useState<LomFieldSchema | undefined>(undefined);
```

Comment:
PERSONAL PREFERENCE: If it is something you set intentianally to empty, i usually use null. And i leave undefined for things that are empty but were not set by me
```suggestion
	const [selected, setSelected] = useState<LomFieldSchema | null>(null);
```

Reply:
Output of `educationLevels.find` can be `undefined` hence this state.